docs/specs/markdown.yml

```diff
@@ -628,15 +628,17 @@ outputs:
 # Liquid template file missing should not crash
```

Comment:
consider move these 2 tests to `static.yml`.

Reply:
move in another PR

---
In reply to: [413546264](https://github.com/dotnet/docfx/pull/5806#discussion_r413546264) [](ancestors = 413546264)